src/components/environment-banner/environment-banner.css

```diff
@@ -0,0 +1,17 @@
+.environment-banner {
+    color: #333;
+}
+
+.environment-banner p {
```

Comment:
These classes actually need to be refactored to not use multiple selectors. `environment-banner p` can be changed to `environment-banner-title` or something similar for example. Thanks!

Reply:
Okay, thanks. I'll work on those changes!